CHANGELOG.md

```diff
@@ -13,10 +13,12 @@ Being a terse compilation by version of changes.
    * Added a `check_constraints()` operator to the `CGPDecoder` class, to help verify custom algorithms
    * Added `LeadingOnes`, `DeceptiveTrap`, and `TwoMax` problems to `binary_rep.problems` module
    * Added `SumPhenotypePlotProbe`, and a new example using it to visualizing MaxOnes-style problems
+   * Added `multiobjective` sub-package that provides support for NSGA-II
+     * `multiobjective.nsga2.nsga2()` top-level monolithic function
+     * `multiobjective.problems.MultiObjectiveProblem` is new abstract base class for multiobjective problems
+     * `multiobjective.ops` contains supporting pipeline operators, though most users will not see those if they use `nsga()`
 
  * API changes
-   * `Problem.evaluate()` now takes an `Individual` instead of a `phenome` as an argument
-     * ^ this is a major API change that is not backword compatible!
```

Comment:
Ope, were these two lines removed on accident?  Perhaps a merge mistake?

Reply:
[This change](https://github.com/AureumChaos/LEAP/pull/223/commits/f16306357a54e12eed71df5813e37899600ecbeb) was made by @markcoletti after he merged NSGA-II back into develop, where I branched off.